mrchecker-framework-modules/mrchecker-database-module/src/resources/settings.properties

```diff
@@ -1 +1,2 @@
-database.lib=./lib/dbdrivers
\ No newline at end of file
+database.lib=./lib/dbdrivers
+core.isEncryptionEnabled=true
```

Comment:
encription  true or false? I don't see you are using encryption, yet

Reply:
Someone might want to use the encryption in future.